pkg/syncthing/monitor.go

```diff
@@ -21,16 +21,20 @@ func (s *Syncthing) IsConnected() bool {
 func (s *Syncthing) Monitor(ctx context.Context, wg *sync.WaitGroup, disconnect chan struct{}) {
 	wg.Add(1)
 	defer wg.Done()
-	ticker := time.NewTicker(5 * time.Second)
+	ticker := time.NewTicker(3 * time.Second)
+	connected := true
 	for {
 		select {
 		case <-ticker.C:
-			if !s.IsConnected() {
-				if !s.IsConnected() {
+			if s.IsConnected() {
+				connected = true
```

Comment:
what is this for? won't the loop exit the first time is not connected anyway?

Reply:
it is to fail for two consecutive connection issues